src/category_theory/functor/currying.lean

```diff
@@ -114,4 +115,14 @@ swapping the factors followed by the uncurrying of `F`. -/
 def uncurry_obj_flip (F : C ⥤ D ⥤ E) : uncurry.obj F.flip ≅ prod.swap _ _ ⋙ uncurry.obj F :=
 nat_iso.of_components (λ p, iso.refl _) (by tidy)
 
+variables (B C D E)
+
+/--
+A version of `category_theory.whiskering_right` for bifunctors, obtained by uncurrying,
+applying `whiskering_right` and currying back
+-/
+@[simps] def whiskering_rigth₂ : (C ⥤ D ⥤ E) ⥤ ((B ⥤ C) ⥤ (B ⥤ D) ⥤ (B ⥤ E)) :=
```

Comment:
```suggestion
@[simps] def whiskering_right₂ : (C ⥤ D ⥤ E) ⥤ ((B ⥤ C) ⥤ (B ⥤ D) ⥤ (B ⥤ E)) :=
```
Are the lemmas generated by simps good/sufficient?

Reply:
I think they should be